src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,176 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private static class Node<T> {
+        T item;
+        private Node<T> next;
+        private Node<T> prev;
+
+        public Node(Node<T> prev, T item, Node<T> next) {
+            this.item = item;
+            this.next = next;
+            this.prev = prev;
+        }
+    }
+
+    private Node<T> first;
+    private Node<T> last;
+    private int size;
+
+    public MyLinkedList() {
+        size = 0;
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        if (size == 0) {
+            first = new Node<>(null, value, null);
+            last = first;
+            size++;
```

Comment:
let's try to remove the repetitive code

create new node in any case that points on `last` as the previous element (it will be null if size is 0)
then if `first` is not set yet, point it on your new node
if it is set, then point `last.next` on your new node

Reply:
Also done